packages/frontend/src/components/SimpleStatistic/SimpleStatistics.styles.tsx

```diff
@@ -1,13 +1,25 @@
 import { Colors } from '@blueprintjs/core';
 import styled from 'styled-components';
 
-export const BigNumberContainer = styled.div`
+interface BigNumberContainerProps {
+    $paddingBottom?: number;
```

Comment:
just a small comment: why is it $paddingBottom and not just paddingBottom? 😅

Reply:
@yasmin-e  if you prefix with dollar sign then the argument will not be passed down to the dom node. see this for
more: https://styled-components.com/docs/api#transient-props